lib/core/viewer/components/viewer.component.ts

```diff
@@ -211,8 +211,8 @@ export class ViewerComponent implements OnChanges, OnInit, OnDestroy {
     urlFileContent: string;
     otherMenu: any;
     extension: string;
-    sidebarRightTemplateContext: { node: Node } = { node: null };
-    sidebarLeftTemplateContext: { node: Node } = { node: null };
+    sidebarRightTemplateContext: { node: Node } = {node: null};
```

Comment:
please restore the indentation changes and setup your ide to leave the space after the parentheses